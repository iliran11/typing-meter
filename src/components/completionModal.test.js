import React from 'react';
import CompletionModal from './completionModal';
import { shallow } from 'enzyme';

const props = {
  correctTypedWords: 78,
  wpmScore: 10,
  onRestart: jest.fn()
}
const wrapper = shallow(<CompletionModal {...props} />);

describe('`CompletionModal`',()=>{
  it ('renders properly',()=>{
    expect(wrapper).toMatchSnapshot();
  })
})