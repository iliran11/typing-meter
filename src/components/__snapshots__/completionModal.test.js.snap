// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`CompletionModal\` renders properly 1`] = `
<Dialog
  autoDetectWindowHeight={true}
  autoScrollBodyContent={false}
  className="completion-dialog"
  modal={false}
  open={true}
  repositionOnUpdate={false}
  title={
    <Title
      wpm={10}
    />
  }
>
  <div
    id="completion-content"
  >
    <h4
      className="size2"
      id="your-score"
    >
      Please Do Your Best!
    </h4>
    <ResultComparison
      result={10}
    />
    <div
      className="score-report"
    >
      <ScoreSection
        iconClass="fas fa-check-circle"
        score={78}
        title="CORRECT"
      />
      <ScoreSection
        iconClass="fas fas fa-tachometer-alt"
        score={10}
        title="WPM"
      />
    </div>
    <h4
      className="size2"
    >
      Try Again?
    </h4>
    <div>
      <div
        className="svg-wrapper svg-size1 restart"
        onClick={[Function]}
      >
        <ActionAutorenew
          style={
            Object {
              "color": null,
              "height": "initial",
            }
          }
        />
      </div>
    </div>
  </div>
</Dialog>
`;
